ts-scripts/fetchIbcDenomTrace.ts

```diff
@@ -32,7 +32,7 @@ export const formatIbcToken = ({
   denom: string
   denomTrace?: IbcDenomTrace
 }): TokenStatic => ({
-  hash: denom.replace('ibc', ''),
+  hash: denom.replace('ibc/', ''),
```

Comment:
don't think we want to keep the `/` in front of the hash. it ended up with some `hash` starting with `/` and some without. i think it's better without, but please let me know if there is some consideration I'm missing